PROJECT_PROGRESSION.md

```diff
@@ -199,43 +199,51 @@ To graduate from At Large stage or for a new project to join as an Impact projec
+It is expected that incubating project will make an active effort to work through the on-boarding process in a reasonable timeframe. They will be offered foundation support to help do so.
 
 #### Acceptance Criteria
 
-Projects may be granted Emeritus status through CPC consensus and with the approval of the maintainers of the project.
-In cases where the project maintainers aren't responding after having been repeatedly contacted through appropriate channels about the CPC's intent to move the project to Emeritus stage, the CPC may proceed with the stage change without approval from the maintainers.
+1. consensus within private mailing list to move into incubation process
 
-### Incubating
+<!--If this checklist is updated please ensure `https://github.com/openjs-foundation/project-status/blob/HEAD/.github/ISSUE_TEMPLATE/02-project-sunsetting-checklist-template.md` is updated as well -->
+### End of Life
 
 #### Definition
 
-Incubating projects are projects that are in the process of completing their on-boarding checklist to join the foundation. Projects in this phase may be looking to join the foundation as At-Large or Impact Stage. Members of incubating projects are invited to join Cross Project Council meetings as a guest, please refer to the [CPC Charter](./CPC-CHARTER.md) for details on how guests may participate in the meetings.
+When an OpenJS software project reaches its end of life, it means that the developers responsible for it will no longer provide support, updates, patches, or enhancements for that version of the software. This includes security updates. The software doesn't disappear; it remains functional but won't receive any further official updates or fixes.
 
-#### Examples
+In most cases, End Of Life (EOL) projects will have a publicly communicated EOL milestone/date, indicating when they will no longer receive support or updates. Once this date is reached, the project should follow the steps outlined in the Archived project checklist.
 
-1. Projects that have been accepted into incubation process after silent period.
+#### Acceptance Criteria
 
-#### Expectations
+Projects may be granted End of Life status through CPC consensus and with the approval of the maintainers of the project.
 
-It is expected that incubating project will make an active effort to work through the on-boarding process in a reasonable timeframe. They will be offered foundation support to help do so.
+In cases where the project maintainers aren't responding after having been repeatedly contacted through appropriate channels about the CPC's intent to move the project to End of Life stage, the CPC may proceed with the stage change without approval from the maintainers.
```

Comment:
Do we have guidelines of what is an "expected" condition of CPC attempting to reach out maintainers? Or is this just based on good judgement?

Reply:
I'd say good judgement for this one. I